### DISCUSSION TITLE
Global Ranking and Country Ranking problem in Profile

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#23081] [Q&A]
First problem I'm gonna tell you is about Global Ranking

I saw a one player that has a lower pp count but has a higher global ranking than mine: https://github.com/ppy/osu/discussions/21831

![211511652-0d32186d-b30d-45b5-ac5a-b2dff8a79f2e](https://user-images.githubusercontent.com/105548541/229331825-08b0b06f-d6a7-4bc8-952e-d42bb8f6e486.png)

As you can see from the picture above, that's his count pp and Global Ranking. Meanwhile, my pp count is higher than him but my Global Ranking is lower than him.
![Screenshot_2023-04-02-12-59-02-226_com opera browser-edit](https://user-images.githubusercontent.com/105548541/229332373-da75d200-fa1d-4036-b825-653d4a3c61b4.jpg)

The second problem is until now my "Country Ranking" is still not updated, it's still empty.

**Comment by Axilotl17** [comment]
are both of these screenshots taken in lazer? 

lazer will display scores from the lazer leaderboard, which is at https://lazer.ppy.sh, while stable will display from the normal leaderboard, https://osu.ppy.sh. if the first screenshot is in lazer, someone with 482 pp might only be 30k, while being a much lower rank in stable.

**Reply by Shirou27** [reply]
It is both lazer which is https://lazer.ppy.sh

**Reply by peppy** [reply]
Currently some rankings will only update when you set a score. It may be that one of the two users has no played recently, and as such has an outdated ranking.

**Reply by Shirou27** [reply]
Is there a fix for that in the future?

I hope it will be fix soon to accurate ranking

**Reply by Shirou27** [reply]
How about the Country Ranking? Will it gonna be fix?
